cloudinit/net/network_state.py

```diff
@@ -379,7 +382,8 @@ def handle_physical(self, command):
             'address': None,
             'gateway': None,
             'subnets': subnets,
-            'accept-ra': accept_ra
+            'accept-ra': accept_ra,
+            'wakeonlan': wakeonlan
```

Comment:
is there a good reason why we don't have a trailing `,` here?

Reply:
Because the original last entry (accept_ra) didn't have a trailing comma ;-)

I can add one if necessary/recommended.

Reply:
Trailing comma added.